<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchGraphs</class>
 <widget class="QWidget" name="BenchGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout" stretch="1,1">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QFrame" name="tempFrame">
       <property name="styleSheet">
        <string notr="true">QFrame
{
	border-radius: 25px;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="tempGraph" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pressureFrame">
       <property name="styleSheet">
        <string notr="true">QFrame
{
	border-radius: 25px;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="pressureGraph" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
